rate.go

```diff
@@ -7,26 +7,27 @@ import (
 
 // A Rate contains information on shipping cost and delivery time.
 type Rate struct {
-	ID                     string     `json:"id,omitempty"`
-	Object                 string     `json:"object,omitempty"`
-	Mode                   string     `json:"mode,omitempty"`
-	CreatedAt              *time.Time `json:"created_at,omitempty"`
-	UpdatedAt              *time.Time `json:"updated_at,omitempty"`
-	Service                string     `json:"service,omitempty"`
-	Carrier                string     `json:"carrier,omitempty"`
-	CarrierAccountID       string     `json:"carrier_account_id,omitempty"`
-	ShipmentID             string     `json:"shipment_id,omitempty"`
-	Rate                   string     `json:"rate,omitempty"`
-	Currency               string     `json:"currency,omitempty"`
-	RetailRate             string     `json:"retail_rate,omitempty"`
-	RetailCurrency         string     `json:"retail_currency,omitempty"`
-	ListRate               string     `json:"list_rate,omitempty"`
-	ListCurrency           string     `json:"list_currency,omitempty"`
-	DeliveryDays           int        `json:"delivery_days,omitempty"`
-	DeliveryDate           *time.Time `json:"delivery_date,omitempty"`
-	DeliveryDateGuaranteed bool       `json:"delivery_date_guaranteed,omitempty"`
-	EstDeliveryDays        int        `json:"est_delivery_days,omitempty"`
-	BillingType            string     `json:"billing_type,omitempty"`
+	ID                     string        `json:"id,omitempty"`
+	Object                 string        `json:"object,omitempty"`
+	Mode                   string        `json:"mode,omitempty"`
+	CreatedAt              *time.Time    `json:"created_at,omitempty"`
+	UpdatedAt              *time.Time    `json:"updated_at,omitempty"`
+	Service                string        `json:"service,omitempty"`
+	Carrier                string        `json:"carrier,omitempty"`
+	CarrierAccountID       string        `json:"carrier_account_id,omitempty"`
+	ShipmentID             string        `json:"shipment_id,omitempty"`
+	Rate                   string        `json:"rate,omitempty"`
+	Currency               string        `json:"currency,omitempty"`
+	RetailRate             string        `json:"retail_rate,omitempty"`
+	RetailCurrency         string        `json:"retail_currency,omitempty"`
+	ListRate               string        `json:"list_rate,omitempty"`
+	ListCurrency           string        `json:"list_currency,omitempty"`
+	DeliveryDays           int           `json:"delivery_days,omitempty"`
+	DeliveryDate           *time.Time    `json:"delivery_date,omitempty"`
+	DeliveryDateGuaranteed bool          `json:"delivery_date_guaranteed,omitempty"`
+	EstDeliveryDays        int           `json:"est_delivery_days,omitempty"`
```

Comment:
What changed between these lines? Tabs to spaces? They shouldn't have been altered... Also gofmt should have complained about this if they changed indentation strategies.

Reply:
I believe it was the indention (and, obviously, the new CarbonOffset property)

Reply:
Right, the indentation should remain the same while the CarbonOffset should be the only alteration here.